### DISCUSSION TITLE
Text not formatted correctly after updates

### DISCUSSION THREAD

**Discussion by lakshadev** [globaleaks/globaleaks-whistleblowing-software#4361] [Q&A] [answered]
I have a channel which has text as follows (in the same format, with breaks)

Following the recent updates, this is how is it being displayed: How do I get the breaks to appear again?

**Comment by evilaliv3** [comment]
Hello @lakshadev ,thank you for notifying this.

 is this the description of the channel or what?

**Reply by lakshadev** [reply]
Hello, yes it is the description of my channels

**Reply by evilaliv3** [reply, marked as answer]
Thank you, the issue should be fixed in release 5.0.41

**Comment by lakshadev** [comment]
Thank you, it worked with the new update, however now my footer is not appearing.
